### DISCUSSION TITLE
tutorials/client-side-graphql-react/03-apollo-sandbox-explorer

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#141] [course feedback]
# tutorials/client-side-graphql-react/03-apollo-sandbox-explorer

https://www.apollographql.com/tutorials/client-side-graphql-react/03-apollo-sandbox-explorer

**Comment by williamphelps13** [comment]
Great tutorial so far! Note that there is 2 sections on this page called "Saving an operation". In addition, I am seeing 2 places to comment,1 of which, as mentioned in a comment on the previous page, is obscured by the side panel.
